sdk/test-utils/recorder/karma.conf.js

```diff
@@ -50,7 +50,7 @@ module.exports = function (config) {
     // inject following environment values into browser testing with window.__env__
     // environment values MUST be exported or set with same console running "karma start"
     // https://www.npmjs.com/package/karma-env-preprocessor
-    envPreprocessor: ["RECORDINGS_RELATIVE_PATH", "PROXY_MANUAL_START", "RECORDING_ASSETS_PATH"],
+    envPreprocessor: ["RECORDINGS_RELATIVE_PATH", "PROXY_MANUAL_START", "RECORDINGS_ASSETS_PATH"],
```

Comment:
why can't we rely on the RECORDINGS_RELATIVE_PATH for RECORDINGS_ASSETS_PATH?

Reply:
The assets.json might not be present, in which case the RECORDING_ASSETS_PATH is left undefined. There's no way of detecting whether it's there in the browser. I experimented with it in the previous PR: https://github.com/Azure/azure-sdk-for-js/pull/23405#pullrequestreview-1179720559. But it didn't work because of that reason.

If we can come up with a better solution I would be thrilled.

Reply:
Hmm, we could catch the error the test proxy throws when the provided assets.json doesn't exist and try again without providing it. That could work! I'll give that a go.

Reply:
> The assets.json might not be present, in which case the RECORDING_ASSETS_PATH is left undefined. There's no way of detecting whether it's there in the browser. I experimented with it in the previous PR: https://github.com/Azure/azure-sdk-for-js/pull/23405#pullrequestreview-1179720559. But it didn't work because of that reason.

oh yeah, that makes sense, I remember that now from Scott's PR.

Reply:
> Hmm, we could catch the error the test proxy throws when the provided assets.json doesn't exist and try again without providing it. That could work! I'll give that a go.

I'm thrilled, but then all the packages have to be migrated at once, right? 
If they don't have the assets.json, they'll fail?

Reply:
The start request would fail initially. We'd look at the failure and if it's because the assets.json doesn't exist we'd make the request again without specifying an assets.json. So it wouldn't be breaking.

Reply:
Beautiful.